agent/logging.go

```diff
@@ -38,39 +36,13 @@ func GetSender(ctx context.Context, prefix, taskId string) (send.Sender, error)
 		senders []send.Sender
 	)
 
-	if os.Getenv(subprocess.MarkerAgentPID) == "" {
```

Comment:
Are you sure we want to get rid of the `MarkerAgentPID` check?

Reply:
It isn't used here anymore because I got rid of the sumologic logger. If we wanted to keep it in I could, but I don't see a reason to

Reply:
I think this means don't set up the splunk logger if the environment variable is empty.

Reply:
fixed